docs/semgrep-secrets/validators.md

```diff
@@ -252,29 +252,28 @@ rules:
   - http:
       request:
         headers:
-          Host: api.exampleCo.com
+          Host: api.semgrep.dev
           User-Agent: Semgrep
         method: POST
         body: |
           {"key": "$REGEX"}
-        url: https://api.exampleCo.com/user
+        url: https://api.semgrep.dev/user
       response:
       - match:
         - status-code: 200
         - content:
             language: regex
             pattern-regex: (\"role\":admin)
-          status-code: 200
-        message: >-
-          The token exposed is for an admin user, this should be fixed immediately! See <insertlink> on how to rotate and look for suspicious activity
-        metadata:
-          context:
-            - admin: true
-        severity: ERROR
         result:
           validity: valid
+          severity: ERROR
+          message: >-
+            The token exposed is for an admin user, this should be fixed immediately! See <insertlink> on how to rotate and look for suspicious activity
```

Comment:
@LewisArdern did you have a specific link in mind for this placeholder?

Reply:
i left it out on purpose

Reply:
Sorry, I'm confused now -- do you want me to take out the links? If so, I think we should leave the message as "The token exposed is for an admin user; this should be fixed immediately!"